test/shopify-cli/js_system_test.rb

```diff
@@ -100,6 +100,13 @@ def test_yarn_check_returns_true_if_yarn_lock_present_and_which_yarn_call_succee
       assert JsSystem.yarn?(@context)
     end
 
+    def test_call_hits_capture3_if_capture_response_is_available
```

Comment:
I feel like this more like `test_call_hits_capture3_if_capture_reponse_set_to_true`